language/llama2-70b/user.conf

```diff
@@ -10,4 +10,11 @@
 *.Server.min_duration = 120000
 *.Server.min_query_count = 100
 
-llama2-70b.Server.sample_concatenate_permutation = 1
\ No newline at end of file
+llama2-70b.Server.sample_concatenate_permutation = 1
+
+# Target Latencies for low latency setting. In order to run low latency:
+# 1. Uncomment this set of latencies
+# 2. Comment the set of latencies in mlperf.conf
+# llama2-70b.Server.target_latency = 0
```

Comment:
I don't think using comment/uncomment is a good way to support low-latency - it is very error-prone for users.

Can we add a new benchmark `llama2-70b-interactive` that inherits all of the `llama2-70b` stuff, but with a new set of latency in the configuration file? The

Reply:
I added a new benchmark called `llama2-70b-low-latency`. Let's decide the name in the WG meeting

Reply:
Update to interactive